### ISSUE TITLE
AutomationProperties not being spoken to users from views (Label, Image, Button, etc) inside of CarouselView on iOS

### ISSUE THREAD

**Issue by tdoell** [alexrainman/CarouselView#488]
**Platform(s): iOS**

**Issue:**
When having any type of **ContentView** inside of the CarouselView, the AutomationProperties for accessibility are not being read to the user on iOS. Android works as expected and reads the text to the user.
**Have not tried UWP**

**Expected:**
When specifying AutomationProperties, the text should be read aloud to the user that needs it for accessibility purposes.

**Example:**
```xaml
<ContentView or Button or Label or Image etc.
    AutomationProperties.HelpText="This is the action"
    AutomationProperties.IsInAccessibleTree="True"
    AutomationProperties.Name="This should be spoken as the phrase"
    BackgroundColor="Green"
    HeightRequest="100"/>
```

On Android, "This should be spoken as the phrase" is spoken to the user correctly.
On iOS, nothing is spoken to the user.

**Steps to Reproduce**
1. Create a blank Xamarin.Forms solution
2. Do the proper steps to set up the CarouselView package
3. Add an ItemsSource with some type of content view
4. Add AutomationProperties to each view in the ItemsSource

I am running Xamarin.Forms 3.4.0.1009999 and the latest version of the **CarouselView** package.

If needed, I can upload a reproduction solution.

**Comment by alexrainman** [comment]
Please, upload a reproduction solution.

**Comment by SravanthiKare** [comment]
@alexrainman 
Even we are facing same issue with CarouselView having ContentView in it.

**Comment by tdoell** [comment]
Here is a reproduction solution for Xamarin.Forms 4.0.0.425677. I didn't test Android in this solution due to not currently having a device to test on. Coworkers using them currently. There is two items in the CarouselView. Both have automation properties on them, but neither of them are spoken.

[CarouselViewBug488.zip](https://github.com/alexrainman/CarouselView/files/3257504/CarouselViewBug488.zip)

**Comment by SravanthiKare** [comment]
@alexrainman 
Can you please help on this regard. This has become a blocker for us to use the component in our application.
